Slim/Schema.pm

```diff
@@ -3293,6 +3300,22 @@ sub canFulltextSearch {
 	return $canFulltextSearch;
 }
 
+sub _workRequired {
+	my $genres = shift;
+	return 1 if !$prefs->get('worksScan');
+	return 0 if $prefs->get('worksScan') == 2;
```

Comment:
Please change the meaning to "off" if 0, partial if 2. It's easier to understand IMHO.